js/deerInitializer.js

```diff
@@ -13,6 +13,7 @@ import { default as DEER } from 'https://centerfordigitalhumanities.github.io/de
 
 // Identify a UTILS package
 import { default as UTILS } from 'https://centerfordigitalhumanities.github.io/deer/releases/alpha-.11/deer-utils.js'
+UTILS.getSafeValue = (property, alsoPeek, asType) => property ? UTILS.getValue(property, alsoPeek, asType) : ""
```

Comment:
The current LRDA data entry doesn't create empty lists, it puts an empty item in each list. That's mostly what I'm trying to avoid here, along with console warns.